src/control/controlmodel.cpp

```diff
@@ -16,11 +16,19 @@ ControlModel::ControlModel(QObject* pParent)
     const QList<QSharedPointer<ControlDoublePrivate>> controlsList =
             ControlDoublePrivate::getAllInstances();
 
+    QSet<ConfigKey> controlKeys;
```

Comment:
ahead of time, predictable construction is probably preferable over doing it in the loop. wdyt?
```suggestion
    const auto aliases = ControlDoublePrivate::getControlAliases();
    QSet<ConfigKey> controlKeys(aliases.cbegin(), aliases.cend()); 
```